src/testInteg/codecatalyst/devEnv.test.ts

```diff
@@ -57,7 +57,7 @@ describe('InactivityMessages', function () {
     })
 
     it('shows expected messages 5 minutes before shutdown on a 15 minute inactivity timeout', async function () {
-        instance.setupMessage(15, devEnvActivity as unknown as DevEnvActivity, relativeMinuteMillis)
+        await instance.setupMessage(15, devEnvActivity as unknown as DevEnvActivity, relativeMinuteMillis)
```

Comment:
The tests in src/testInteg/codecatalyst/devEnv.test.ts take 19s to run on both `master` and in this PR.